engine/inline_rules_normalizer.go

```diff
@@ -71,6 +72,11 @@ func processWorkflow(workflow PadWorkflow, currentRules []PadRule) (*PadWorkflow
 		return nil, nil, err
 	}
 
+	// the run property was used
+	if len(runs) > 0 {
+		wf.RunUsed = true
+	}
```

Comment:
Why don't we use the property `AlwaysRun` for this? 

```suggestion
	// To provide backward compatibility we assume that
	// having the run property is the same as always-run.
	if len(runs) > 0 {
		wf. AlwaysRun = true
	}
```